test/test_core.py

```diff
@@ -61,9 +66,61 @@ def test_import_julia_functions(self):
         else:
             pass
 
+    def test_import_julia_module_existing_function(self):
+        from julia import Base
+        assert Base.mod(2, 2) == 0
+
+    def test_import_julia_module_non_existing_name(self):
+        from julia import Base
+        try:
+            Base.spamspamspam
+            self.fail('No AttributeError')
+        except AttributeError:
+            pass
+
+    def test_julia_module_bang(self):
+        from julia import Base
+        xs = [1, 2, 3]
+        ys = Base.scale_b(xs[:], 2)
+        assert all(x * 2 == y for x, y in zip(xs, ys))
+
+    def test_import_julia_submodule(self):
+        from julia.Base import REPL
+        assert isinstance(REPL, ModuleType)
+
+    def test_star_import_julia_module(self):
+        from . import _star_import
+        _star_import.BasicREPL
+
+    def test_main_module(self):
+        from julia import Main
+        Main.x = x = 123456
+        assert julia.eval('x') == x
+
+    def test_module_all(self):
+        from julia import Base
+        assert 'resize_b' in Base.__all__
+
+    def test_module_dir(self):
+        from julia import Base
+        assert 'resize_b' in dir(Base)
+
+    def test_import_without_setup(self):
+        command = [sys.executable, '-c', 'from julia import Base']
+        print('Executing:', *command)
+        subprocess.check_call(command, env=_orig_env)
```

Comment:
`env=_orig_env` probably is required to fix https://travis-ci.org/JuliaPy/pyjulia/jobs/379514563#L298

Reply:
Yes it looks like it fixes the failure: https://travis-ci.org/JuliaPy/pyjulia/builds/379524930